### ISSUE TITLE
Can you please add a placeholder that i can use it for a chat plugin.

### ISSUE THREAD

**Issue by ThomasDeStrooper** [OGminso/ChatHeadFont#7]
Im using another chat plugin and i whas hoping that you couldm ake a placeholder for the player heads so that i can use it in another chat plugin.

**Comment by TerdyTheTerd** [comment]
Yeah this would be the best improvement to make, as is all chat formatting gets replaced with just the default chat formatting.
